enterprise/server-enterprise/pom.xml

```diff
@@ -257,6 +257,26 @@
         </configuration>
       </plugin>
 
+      <plugin>
```

Comment:
See eg. the kernel module for what this is - this generates asciidoc from the specified Settings files